### ISSUE TITLE
Issue about the training script

### ISSUE THREAD

**Issue by LoveSiameseCat** [HighwayWu/ImageForensicsOSN#7]
Hello,

Thank you for offering the training code. However, this version misses many parameters, such as learning rate. could you check it, and give a revised version.

**Comment by HighwayWu** [comment]
Thanks for your kind reminder, I will check and update it.

**Comment by LoveSiameseCat** [comment]
Thank you for your quick response. I will wait for your update. By the way, I wanted to say that your work in the forensics field is impressive. If it's convenient for you, could I add you on WeChat?

**Comment by HighwayWu** [comment]
Sure, welcome to have more discussion on forensics field. Please send an email to me (yc07912@um.edu.mo) for adding the WeChat.

**Comment by LoveSiameseCat** [comment]
Okay, I have sent an email to you from my school email.&nbsp;
&nbsp;


Anwei&nbsp;luo
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月13日(星期四) 中午12:08
收件人: ***@***.***&gt;; 
抄送: "Anwei ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [HighwayWu/ImageForensicsOSN] Issue about the training script (Issue #7)





 
Sure, welcome to have more discussion on forensics field. Please send an email to me ***@***.***) for adding the WeChat.
